examples/scanner.py

```diff
@@ -0,0 +1,20 @@
+# Used to find adress of Sky Fan
+from bluepy.btle import Scanner, DefaultDelegate
+
+class ScanDelegate(DefaultDelegate):
+    def __init__(self):
+        DefaultDelegate.__init__(self)
+
+    def handleDiscovery(self, dev, isNewDev, isNewData):
+        if isNewDev:
+            print ("Discovered device", dev.addr)
+        elif isNewData:
+            print ("Received new data from", dev.addr)
+
+scanner = Scanner().withDelegate(ScanDelegate())
+devices = scanner.scan(10.0)
```

Comment:
Shouldn't there be a simple filter or something to remove most of the devices? My device's name is `Intellivent Sky`.

Reply:
Would probably be nice maby some more tests before applying filter ?